doc/general/operation-modes/operation-modes.md

```diff
@@ -106,58 +108,36 @@ For the operation of the installed components of the fiskaltrust.Middleware foll
+#### Launcher
 
-#### Configuration Scenarios
+The Launcher is a software (file) named `fiskaltrust.exe`, which is used only for the on-premise installed products (e.g. AT products fiskaltrust.SignatureCard or fiskaltrust.SignatureBox). For Windows, it is a .NET command-line application and a .NET Windows service. For Linux and Mac, the launcher can be executed via Mono, version 3.2.8 or higher, or used as daemon.
 
-The POS System connects to the fiskaltrust.Middleware to process the receipt chaining calculation. A Cash register means an individual fiskaltrust.SecurityMechanism - in the fiskaltrust.Portal it is called "CashBox" and represents the real CashRegister and its configuration with the SecurityMechanism. The cash register is identified via the ftCashBoxId. This is unique worldwide and also the first part of the authentication on the fiskaltrust.Helipad. The second part is an authentication token.
+The main tasks of the launcher are:
 
-#### Single Queue scenario
+  - providing basic configuration settings such as ftCashBoxId and access token
+  - comparing the configuration data retrieved from fiskaltrust.Helipad with the local configuration
+  - updating queue- and SCU packages accordingly to the configuration
+  - execution of configured packages
+  - load balancing of multiple queues
 
-The most common scenario uses a connection to a single fiskaltrust.Queue, which takes care of performing all the requested operations in accordance to the national law and requlations.
+The executable file `fiskaltrust.exe` and the corresponding DLLs can be distributed via copy-paste and then configured and installed with the help of a command-line parameter. It can be downloaded (incl. configuration) from the Portal’s configuration->cashbox page, or found on nuget.org and configured manually.
 
-![](./images/01-individual-queue.png)
+Call parameter of the launcher are documented in the chapter [installation](../installation/installation.md).
 
-<span id="_Toc527986809" class="anchor"></span>*Illustration* *8. Use of an individual queue.*
+### Private cloud (operated by a third party) installed components
 
-#### Scenario with several queues for performance improvement
+No installation or configuration is required for the client and any platform can use this service. The availability and use of SaaS installed components is dependent on local regulations and currently available for Austria and Germany. In Germany, because of different market regulation, the fiskaltrust.Middleware components must be hosted and operated by the POS creator in the same datatcenter where the receipt generation process of a distributed point of sale system occurs.
```

Comment:
```suggestion
No installation or configuration is required for the client and any platform can use this service. The availability and use of SaaS installed components is dependent on local regulations and currently available for Austria and Germany. In Germany, because of different market regulation, the fiskaltrust.Middleware components must be hosted and operated by the PosCreator in the same datacenter where the receipt generation process of a distributed point of sale system occurs.
```